src/async_wrap-inl.h

```diff
@@ -50,35 +61,34 @@ inline double AsyncWrap::get_trigger_async_id() const {
 }
 
 
-inline v8::MaybeLocal<v8::Value> AsyncWrap::MakeCallback(
-    const v8::Local<v8::String> symbol,
+inline MaybeLocal<Value> AsyncWrap::MakeCallback(
+    const Local<String> symbol,
     int argc,
-    v8::Local<v8::Value>* argv) {
-  return MakeCallback(symbol.As<v8::Name>(), argc, argv);
+    Local<Value>* argv) {
+  return MakeCallback(symbol.As<Name>(), argc, argv);
 }
 
 
-inline v8::MaybeLocal<v8::Value> AsyncWrap::MakeCallback(
-    const v8::Local<v8::Symbol> symbol,
+inline MaybeLocal<Value> AsyncWrap::MakeCallback(
+    const Local<Symbol> symbol,
     int argc,
-    v8::Local<v8::Value>* argv) {
-  return MakeCallback(symbol.As<v8::Name>(), argc, argv);
+    Local<Value>* argv) {
+  return MakeCallback(symbol.As<Name>(), argc, argv);
 }
 
 
-inline v8::MaybeLocal<v8::Value> AsyncWrap::MakeCallback(
-    const v8::Local<v8::Name> symbol,
+inline MaybeLocal<Value> AsyncWrap::MakeCallback(
+    const Local<Name> symbol,
     int argc,
-    v8::Local<v8::Value>* argv) {
-  v8::Local<v8::Value> cb_v;
+    Local<Value>* argv) {
+  Local<Value> cb_v;
   if (!object()->Get(env()->context(), symbol).ToLocal(&cb_v))
-    return v8::MaybeLocal<v8::Value>();
+    return MaybeLocal<Value>();
   if (!cb_v->IsFunction()) {
-    // TODO(addaleax): We should throw an error here to fulfill the
-    // `MaybeLocal<>` API contract.
-    return v8::MaybeLocal<v8::Value>();
+    Isolate* isolate = env()->context()->GetIsolate();
```

Comment:
```suggestion
    Isolate* isolate = env()->isolate();
```

Reply:
@addaleax, thanks. made the suggested change.